C/Checkpoints/Package.toml

```diff
@@ -1,3 +1,3 @@
 name = "Checkpoints"
 uuid = "b4a3413d-e481-5afc-88ff-bdfbd6a50dce"
-repo = "https://gitlab.invenia.ca/invenia/Checkpoints.jl.git"
+repo = "https://github.com/invenia/Checkpoints.jl"
```

Comment:
Was removing `.git` suffix international?

Reply:
It was not
```suggestion
repo = "https://github.com/invenia/Checkpoints.jl.git"
```